File: fixtures/scale/csharp/large/Abstractions/BenchSample.cs
namespace CipherLab.Abstractions
{
    public class BenchSample
    {
        public string Family { get; }
        public long Nanoseconds { get; }
        public int Iterations { get; }

        public BenchSample(string family, long nanoseconds, int iterations)
        {
            Family = family;
            Nanoseconds = nanoseconds;
            Iterations = iterations;
        }

        public double PerOp()
        {
            return Iterations == 0 ? 0 : (double)Nanoseconds / Iterations;
        }
    }
}

File: fixtures/scale/php/large/src/Abstractions/BenchSample.php
<?php

namespace CipherLab\Abstractions;

class BenchSample
{
    public function __construct(public string $family, public float $nanoseconds, public int $iterations)
    {
    }

    public function perOp(): float
    {
        return $this->iterations === 0 ? 0.0 : $this->nanoseconds / $this->iterations;
    }
}
